src/software/thunderscope/robot_communication.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         estop_mode: EstopMode,
         estop_path: os.PathLike = None,
         estop_baudrate: int = 115200,
+        enable_radio: bool = False,
```

Comment:
I would suggest making a enum class that goes over the different types of comms, and using that instead of a boolean for the argument to constructor and class field. Its fine for now since this PR is priority, so let's make an issue, and it can be a great starter ticket for another member to learn this pretty complicated class.